samples/sensor/mcux_acmp/boards/mimxrt1170_evk_cm7.overlay

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
can these configurations move to the boards folder?

Reply:
Same as above.